tests/decorators/decorators.go

```diff
@@ -5,6 +5,7 @@ import . "github.com/onsi/ginkgo/v2"
 var (
 	//
 	Quarantine = []interface{}{Label("QUARANTINE")}
+	Periodic   = []interface{}{Label("PERIODIC")}
```

Comment:
Future enhancement: Can we add a comment or documentation clearly explaining that tests with this label will only run on periodic lanes? To me, it is not clear that tests with this label will be skipped on the pre-submit lanes, instead, it looks like it identifies the only tests that will be performed by periodic lanes.
WDYT?

Reply:
Good point - renaming the label towards `PERIODIC_ONLY` might make it more clear. WDYT @fossedihelm  ?

Reply:
It could help, but I still think that a comment, like for `RequiresNoSnapshotStorageClass` for example, could be enough. I mean, we can change the label towards, but it will not be enough from my point of view. Just my 2 cents :)

Reply:
That's true :thinking:

Reply:
Comments tend to lie over time ;-)